testsuite/diem-fuzzer/src/fuzz_targets.rs

```diff
@@ -67,10 +66,6 @@ static ALL_TARGETS: Lazy<BTreeMap<&'static str, Box<dyn FuzzTargetImpl>>> = Lazy
         Box::new(safety_rules::SafetyRulesHandleMessage::default()),
         Box::new(safety_rules::SafetyRulesSignProposal::default()),
         Box::new(safety_rules::SafetyRulesSignTimeout::default()),
-        // Secure JSON RPC Client
```

Comment:
Yeah, this isn't super great, but perhaps we could think about fuzzing for this new client going forward.

Reply:
No use fuzzing things that aren't used :D 

But in all seriousness I do plan on adding more tests/fuzzing going forward. The plan was to get folks to agree to this before I spent time doing more work.